src/main/java/model/GetAllUserProfileDetailsRequest.java

```diff
@@ -0,0 +1,26 @@
+package model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Map;
+
+@NoArgsConstructor
+@Data
+@Builder
+@AllArgsConstructor
+public class GetAllUserProfileDetailsRequest {
```

Comment:
This should be named GetUserProfileRequest should be similar to the lambda name (GetUserProfileLambda)